palisade-service/README.md

```diff
@@ -34,8 +41,7 @@ to the Filtered Resource Service and retrieve the processed request.
 
 (fields marked with * are acquired from headers metadata)
 
-The service accepts a `PalisdeRequest`, creates a token to return to the client as a `PalisadeResponse`, the Palisade Service then takes the user, resourceId and context from the `PalisadeRequest`, and sends them on to the User Service for further
-processing.
+The service accepts a `PalisdeRequest`, creates a token to return to the client as a `PalisadeResponse`, the Palisade Service then takes the user, resourceId and context from the `PalisadeRequest`, and sends them on to the User Service for further processing.
```

Comment:
Again, I think that the mention of User Service could be replaced with a mention of a chain/set/series of services.

Reply:
The root palisade services readme should encoumpace more of the overview, individual services should really only mention the service they recieve something from, what they do, and what they send to the next service